### ISSUE TITLE
Returning Astropy Table instead of Pandas DataFrame when using search_metadata() in JSOCClient.

### ISSUE THREAD

**Issue by DShivansh** [sunpy/sunpy#3941]
In JSOCClient when we use search metadata like ->
```python
metadata = client.search_metadata(time, series, wavelength) #this returns pandas dataframe 

print(metadata[['T_OBS', 'WAVELNTH']]) #this filters the result with the columns mentioned
```
I think this inconsistency should be removed because even the ``_lookup_records`` in **jsoc.py** returns Astropy Table when this function is not called by ``search_metadata``. Astropy Tables have matured enough to support same syntax as pandas dataframe to filter the columns.

**Comment by sashank27** [comment]
Related issue: #3293

**Comment by nikita-0209** [comment]
Hi. I am a novice. Is anyone still working on this issue? If not, can I begin working on it?

**Comment by nabobalis** [comment]
Hi @nikita-0209, we do have an open pull request that is tackling this issue: https://github.com/sunpy/sunpy/pull/3950 

I would suggest trying another issue.

**Comment by nabobalis** [comment]
Most likely we will be deleting this method in future, so changing the return is not necessary.

**Comment by kakirastern** [comment]
To be closed automatically once #4358 is merged...
